src/dispatch/case/service.py

```diff
@@ -187,6 +189,24 @@ def create(*, db_session, case_in: CaseCreate, current_user: DispatchUser = None
         case_id=case.id,
     )
 
+    # add reporter
```

Comment:
We also need to ensure that `case.assignee` is a proper participant e.g. https://github.com/Netflix/dispatch/pull/2800/files#diff-85562e526dc8615124aead4d1b1dc02d9fbfdbc211396e4c880f3e63f354fb00R156

We will likely need some data migration for that as well (e.g. moving from email to participant_role_id).

Reply:
Addressed here https://github.com/Netflix/dispatch/pull/2800/commits/f5d778bcb5b91abae8b87ea64af7ab08c67e89e0 will need to a do a data migration for existing cases though.